src/app/Task3/task3.js

```diff
@@ -0,0 +1,18 @@
+// Recursion
+// We’ve seen that % (the remainder operator) can be used to test whether a
+// number is even or odd by using % 2 to check whether it’s divisible by two.
+// Here’s another way to define whether a positive whole number is even or odd:
+// • Zero is even.
+// • One is odd.
+// • For any other number N, its evenness is the same as N - 2.
+// Define a recursive function isEven corresponding to this description. The function should accept
+//a number parameter and return a Boolean.
+
+// Test it on 50 and 75. See how it behaves on -1. Why? Can you think of a
+// way to fix this?
+
+function isEven(number) {
+  return Math.abs(number) < 2
+    ? !Math.abs(number)
+    : isEven(Math.abs(number) - 2);
```

Comment:
Actually 1 time is enough for `Math.abs()` to be called. Just declare new variable (not to reassign the passed one), put absolute value there and use it for return statement.